index.html

```diff
@@ -1,157 +1,190 @@
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <div class="tags">
                         <h6>Ruby on rails</h6>
-                        <h6>RSpec</h6>
-                        <h6>Capybara</h6>
-                        <h6>Selenium</h6>
-                    </div>
-                </li>
-                <li class="about-myself-items">
-                    <img src="Images/icon-skills.svg" alt="icon for skills">
-                    <h4>Skills</h4>
-                    <div class="tags-about-me">
-                        <h6>Codekit</h6>
-                        <h6>GitHub</h6>
-                        <h6>Codepen</h6>
-                        <h6>Gitlab</h6>
-                        <h6>Terminal</h6>
+                        <h6>CSS</h6>
+                        <h6>JavaScript</h6>
+                        <h6>HTML</h6>
                     </div>
-                </li>
-            </ul>
-        </section>
-    </body>
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- about myself section -->
+    <section class="about-myself">
+        <h1 class="primary-text primary-text-about-myself">About me</h1>
+        <p class="supporting-text-about-me"> I can help you build a product , feature
+            or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+        <a href="#" class="button button-about-me">Get My Resume</a>
+        <ul class="categories">
+            <li class="about-myself-items">
+                <img src="Images/icon-languages.svg" alt="icon for languages">
+                <h4>Languages</h4>
+                <div class="tags-about-me">
+                    <h6>JavaScript</h6>
+                    <h6>Ruby</h6>
+                    <h6>HTML</h6>
+                    <h6>CSS</h6>
+                </div>
+            </li>
+            <li class="about-myself-items">
+                <img src="Images/icon-frameworks.svg" alt="icon for frameworks">
+                <h4>Frameworks</h4>
+                <div class="tags-about-me">
+                    <h6>Bootstrap</h6>
+                    <h6>Ruby on rails</h6>
+                    <h6>RSpec</h6>
+                    <h6>Capybara</h6>
+                    <h6>Selenium</h6>
+                </div>
+            </li>
+            <li class="about-myself-items">
+                <img src="Images/icon-skills.svg" alt="icon for skills">
+                <h4>Skills</h4>
+                <div class="tags-about-me">
+                    <h6>Codekit</h6>
+                    <h6>GitHub</h6>
+                    <h6>Codepen</h6>
+                    <h6>Gitlab</h6>
+                    <h6>Terminal</h6>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <!-- Form section -->
+    <section class="form">
```

Comment:
- [ ] [OPTIONAL] Hi @Sahar-AbdelSamad, good job working on the project so far 💪🏻!  I think that changing this section form container from a `section` to a `footer` would improve the readability of your code and also conform to using semantic HTML. What do you think 😏?

Reply:
Thank you for your Kind review!